src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react';
+import WorldBeersComponent from './screens/WorldBeers/WorldBeers';
+import BeerOfTheDayComponent from './screens/BeerOfTheDay/BeerOfTheDay';
+import FavouriteBeersComponent from './screens/BeersFavourite/FavouriteBeers';
+import SearchBeersComponent from './screens/SearchBeers/SearchBeers';
+import Navigation from './shared/components/Navigation/Navigation';
+import './App.scss';
+
+const pagesOrScreens = [
```

Comment:
So pages or screens? Names must be clear.

Reply:
Done!